core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -462,13 +463,14 @@ class Invoker(
      */
     private def makeWhiskActivation(
         msg: Message,
-        actionName: EntityName,
+        actionName: EntityPath,
         actionVersion: SemVer,
         activationResponse: ActivationResponse,
-        interval: Interval) = {
+        interval: Interval,
+        limits: Option[ActionLimits]) = {
         WhiskActivation(
             namespace = msg.subject.namespace,
-            name = actionName,
+            name = actionName.last,
```

Comment:
we agree that in the future, name will be the fully qualified name but this is an invasive change to make now and deferring it; as such an annotation is added to contain the fully qualified name as an intermediate point.